vmdb/product/toolbars/ems_infra_center_tb.yaml

```diff
@@ -22,6 +22,11 @@
       :text: 'Edit this #{ui_lookup(:table=>"ems_infra")}'
       :title: 'Edit this #{ui_lookup(:table=>"ems_infra")}'
       :url: '/edit'
+    - :button: ems_infra_scale
+      :image: edit
+      :text: 'Scale this #{ui_lookup(:table=>"ems_infra")}'
+      :title: 'Scale this #{ui_lookup(:table=>"ems_infra")}'
```

Comment:
so you will need to rebase this with master and change it to <%= ui_lookup(:table=>"ems_infra") %>